microcloud/cmd/microcloud/ask.go

```diff
@@ -483,21 +483,44 @@ func askRemotePool(systems map[string]InitSystem, autoSetup bool, wipeAllDisks b
 	return nil
 }
 
-func askNetwork(sh *service.Handler, peers map[string]mdns.ServerInfo, lxdConfig map[string][]api.ClusterMemberConfigKey, bootstrap bool, autoSetup bool) (map[string]string, map[string]string, error) {
```

Comment:
This function signature is much nicer :)

Reply:
I very much agree :)